packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -342,7 +342,9 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     }
   }
 
-  data.assignAllTeamMembers = assignAllTeamMembers ?? false;
+  if (typeof assignAllTeamMembers) {
```

Comment:
```suggestion
  if (assignAllTeamMembers !== undefined) {
```

This would be better imo